MaxText/multimodal_utils.py

```diff
@@ -239,12 +258,12 @@ def pad_to_best_fit_jax(
     dest_slicer_dims.append(slice(None))  # Ellipsis for batch dimensions
   dest_slicer_dims.extend([slice(0, h_to_copy), slice(0, w_to_copy), slice(None)])
 
-  padded_output = padded_output.at[tuple(dest_slicer_dims)].set(image_data_to_place)
+  padded_output[tuple(dest_slicer_dims)] = image_data_to_place
 
   return padded_output
 
 
-def split_to_tiles_jax(images: jnp.ndarray, num_tiles_height: int, num_tiles_width: int) -> jnp.ndarray:
+def split_to_tiles_jax(images: np.ndarray, num_tiles_height: int, num_tiles_width: int) -> np.ndarray:
   """
```

Comment:
why jnp -> np change was needed?

if it's needed should we rename the function to split_to_tiles_jax?

Reply:
The jnp -> np changes are made on data pre-processing functions that are intended to run on CPU, when TPU is used for training. Jax functions will try to access TPU and results in error.